### DISCUSSION TITLE
Fast mode amd64 on m1 mac

### DISCUSSION THREAD

**Discussion by ajaya** [lima-vm/lima#922] [Q&A] [answered]
Folks

I was following the instructions at 
https://github.com/lima-vm/lima/blob/master/docs/multi-arch.md#fast-mode

It talks about upgrading the VM but then talks about running containers.  I am super confused.   So in fast mode there isn't a way to run a amd64 VM? 

AJ

**Comment by jandubois** [comment, marked as answer]
> It talks about upgrading the VM but then talks about running containers. I am super confused. So in fast mode there isn't a way to run a amd64 VM?

No, fast mode means the VM runs with the native instruction set, and only the container code is being emulated. Slow mode means the whole VM is emulated, so everything is slow...

**Reply by ajaya** [reply]
Thank you. That makes sense.
